web/Controllers/LandlordController.cs

```diff
@@ -77,19 +78,16 @@ public async Task<ActionResult> RegisterPost([FromForm] LandlordProfileModel cre
         {
             case ILandlordService.LandlordRegistrationResult.Success:
                 _logger.LogInformation("Successfully created landlord for user {UserId}", user.Id);
-                var msgBody = "A Landlord has just registered\n"
+                var msgBody = $"A Landlord has just registered\n"
                               + "\n"
                               + $"Title: {createModel.Title}\n"
                               + $"First Name: {createModel.FirstName}" + "\n"
                               + $"Last Name: {createModel.LastName}" + "\n"
                               + $"Company Name: {createModel.CompanyName}" + "\n"
                               + $"Email: {createModel.Email}" + "\n"
                               + $"Phone: {createModel.Phone}" + "\n";
-                _mailService.SendMsg(msgBody);
-                if (!createModel.Unassigned)
-                {
-                    return RedirectToAction("MyProfile");
-                }
+                var subject = "Bricks&Hearts - landlord registration notification";
+                Task.Run( () => _mailService.TrySendMsg(msgBody, subject));

```

Comment:
You can disable this warning with a comment, Rider should give you a prompt to do this
```suggestion
#pragma warning disable CS4014
                Task.Run(() => _mailService.TrySendMsg(msgBody, subject));
#pragma warning restore CS4014
```